src/FixturesKernel.php

```diff
@@ -60,8 +61,9 @@ private function prepareSession(Request $request): void
 
         $cookies = $request->cookies;
 
-        if ($cookies->has($session->getName())) {
-            $session->setId($cookies->get($session->getName()));
+        $value = $cookies->get($session->getName());
+        if ($value !== null) {
```

Comment:
Please replace `if ($value !== null) {` with `if (is_string($value)) {`.

---

According to the `\Symfony\Component\HttpFoundation\InputBag::get` method DocBlock it can return 5 different data types (the `null` is one of them), but `\Symfony\Component\HttpFoundation\Session\Session::setId` only accepts a `string`.

P.S.
I wonder why PhpStan isn't complaining about this code.